controllers/user.js

```diff
@@ -66,22 +69,22 @@ class Users {
     * @param {object} res - The response object.
     * @param {object} next - The response object.
     */
-  static login(req, res, next) {
-    passport.authenticate('local', { session: false }, (
-      err,
-      user
-    ) => {
-      if (err) {
-        return next(err);
-      }
-
-      if (user) {
-        return res.json({ message: 'Login was successful' });
-      }
-      return res.status(401).json('incorrect email or password');
+  static async login(req, res) {
+    const { usernameOrEmail, password } = req.body;
+    const loginUser = await User
+      .findOne({
+        where: {
+          [or]: [{ username: { [iLike]: usernameOrEmail } },
```

Comment:
Step the `username` object to the next line with its curly braces so that it aligns with the email object.